app/controllers/devise_token_auth/registrations_controller.rb

```diff
@@ -30,11 +30,6 @@ def create
       # if whitelist is set, validate redirect_url against whitelist
       return render_create_error_redirect_url_not_allowed if blacklisted_redirect_url?(@redirect_url)
 
-      # override email confirmation, must be sent manually from ctrl
-      callback_name = defined?(ActiveRecord) && resource_class < ActiveRecord::Base ? :commit : :create
-      resource_class.set_callback(callback_name, :after, :send_on_create_confirmation_instructions)
-      resource_class.skip_callback(callback_name, :after, :send_on_create_confirmation_instructions)
-
```

Comment:
[Original code](https://github.com/heartcombo/devise/blob/eed641d2bea11839ab13e943660da41cad14314d/lib/devise/models/confirmable.rb#L51-L57) only works if `send_confirmation_notification?` is `true`.
This gem has already set `@skip_confirmation_notification = true`, so we don't need this code.

https://github.com/lynndylanhurley/devise_token_auth/blob/5ef7197d5e2ca50f7fc92a7b04884341759b6e87/app/controllers/devise_token_auth/registrations_controller.rb#L38-L41